env.example

```diff
@@ -338,3 +341,40 @@ RESTART_POLICY=unless-stopped
 
 # Authenticate using external service or just focus external auth window if there is one already.
 # TOKEN_AUTH_URL=https://auth.meet.example.com/{room}
+
+# Jicofo config
+JICOFO_SHORT_ID=1
```

Comment:
nit: I would name it JICOFO_OCTO_ID

Reply:
I thought of it but I don't want to break the earlier functionality like everyone have to change this variable too. Just upgrading to new image won't be enough. That's why I didn't change it.